dspace-spring-rest/src/test/java/org/dspace/app/rest/model/SearchConfigurationRestTest.java

```diff
@@ -0,0 +1,219 @@
+package org.dspace.app.rest.model;
+
+import org.dspace.app.rest.DiscoveryRestController;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by raf on 26/09/2017.
+ */
+public class SearchConfigurationRestTest {
```

Comment:
Again, only test stuff that is "complex" and might break. Unless your getter or setters does something special, you don't need to test it.